### DISCUSSION TITLE
RUC sounding Op40 analysis - determine neighbooring gridpoints?

### DISCUSSION THREAD

**Discussion by aeharding** [NOAA-GSL/GSL-Regional-Model-Forum#31] [Q&A] [answered]
Hello!

I was wondering if there was a way to calculate, or fetch, the neighboring gridpoints for an Op40 analysis.

I have a use case where https://rucsoundings.noaa.gov/ will give me the nearest gridpoint located in the mountains, when a gridpoint that's a bit further away, but not in the mountains, may be more useful.

Thanks!
Alex

**Comment by Brian-Jamison** [comment, marked as answer]
Hello Alex,

The grid for Op40 has a grid spacing of 40km, but I don't have a file of the lat,lon of each of the grid points.  However, on the rucsoundings page, you can select the Ascii Text option which will give the lat,lon of that site in line 1 (below the CAPE line).  The longitude is in degrees West, but doesn't include the minus sign in this format.  Using this, you can go back and input the lat,lon in the Site(s) box.  Be sure to include the minus sign for longitude and input the value as lat,-lon with one comma and no spaces.  Now, you can "nudge" the location in any direction by adjusting the location lat and lon values.  Let me know if you have any problems.

Thanks,

Brian
